src/components/Footer/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import './Footer.css';
+import LOGO from '../../assets/img/craneLogo.png';
+
+const Footer = () => (
+  <React.Fragment>
+    <footer className="footer">
+      <div className="footer-inner">
+        <div className="logo">
+          <a href="https://cranecloud.io">
+            <img
+              src={LOGO}
+              alt="Logo"
+              className="logo-img"
+            />
+          </a>
+        </div>
+
+        <div className="footer-address">
+          East Africa
+          <br />
+          +256(782)75 7010
+          <br />
+          COCIS Building Block B
+          <br />
+          Pool Road, Kampala
+          <br />
+          Makerere University
+          <br />
+          Kampala, Uganda
+        </div>
+
+        <div className="footer-address">
+          {/* <Link to="/">Home</Link> */}Home
+          <br />
+          {/* <Link to="/about">About Us</Link> */}About
+          <br />
+          {/* <Link to="/contact">Contact Us</Link> */}Contact
+          <br />
+          {/* <Link to="news">News & Insights</Link> */}News
+          <br />
+          {/* <Link to="/resources">Resources</Link> */}Resource
+          <br />
+          {/* <Link to="/team">Team</Link> */}Team
+        </div>
+
+        <div className="footer-social">
+          <a
+            className="social-twitter"
+            href="https://twitter.com/cranecloud"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <i className="fa fa-twitter fa-3x"></i>
+          </a>
+
+          <a
+            className="social-linkedin"
+            href="https://www.linkedin.com/company/cranecloud"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <i className="fa fa-linkedin fa-3x"></i>
+          </a>
+        </div>
+
+        <div className="footer-menu">
+          <ul id="menu-footer" className="menu">
```

Comment:
use a better data structure for this - a list maybe - so that you don't hard code every list item here

Reply:
@ckwagaba kindly look again, this is a LIST, unordered list to be specific.